backend/subletting/models.py

```diff
@@ -0,0 +1,53 @@
+from django.contrib.auth import get_user_model
+from django.db import models
+from django.utils import timezone
+
+
+User = get_user_model()
+
+class Offer(models.Model):
+    user = models.ForeignKey(User, on_delete=models.CASCADE, related_name="offers_made")
+    sublet = models.ForeignKey("Sublet", on_delete=models.CASCADE, related_name="offers")
+    #TODO: check if the way we're handling phone/email is fine, validate on frontend/routes that at least one is not null
+    email = models.EmailField(max_length=255, null=True)
+    phone_number = models.CharField(max_length=20, null=True)
```

Comment:
Validation here might be tricky. We can either
1. Validate phone number in serializer
2. Import a library to do it
For simplicity I think option 1 is fine

Reply:
any reason not to use a PhoneNumberField like you suggested?

Reply:
@vcai122 I don't think django has this field by default. We would need to import